pkg/virt-handler/notify-server/server.go

```diff
@@ -84,6 +84,7 @@ func (n *Notify) HandleDomainEvent(ctx context.Context, request *notifyv1.Domain
 	case string(watch.Deleted):
 		n.EventChan <- watch.Event{Type: watch.Deleted, Object: domain}
 	case string(watch.Error):
+		log.Log.Object(domain).Errorf("Domain error event with message: %s", status.Message)
```

Comment:
i think in case of an error message we have just status struct and not domain
but anyway i will take care of it in the changes of #3806 or following